sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/Data.java

```diff
@@ -233,6 +288,23 @@ public static void setDataStoreRemoteOperationListener(DataStoreEventListener li
         getInstance().mEventListener = listener;
     }
 
+    @Nullable
+    private static JSONObject convertToJSONObject(String document){
+        JSONObject jsonObject = null;
+        try {
+            jsonObject = new JSONObject(document);
+        } catch (JSONException e) {
+            AppCenterLog.debug(LOG_TAG, "Can not convert to JSONObject");
```

Comment:
Please keep the exception message

```suggestion
            AppCenterLog.debug(LOG_TAG, "Can not convert to JSONObject", e);
```